Makefile

```diff
@@ -0,0 +1,60 @@
+.DEFAULT_GOAL=all
+.PHONY: clean all copy_env_files lemur_checkout build_containers restart_containers inspect_image_size
+
+# Shell to use for running scripts
+SHELL := $(shell which bash)
+# Get docker path or an empty string
+DOCKER := $(shell command -v docker)
+# Get docker-compose path or an empty string
+DOCKER_COMPOSE := $(shell command -v docker-compose)
+
+# Test if the dependencies we need to run this Makefile are installed
+deps:
+ifndef DOCKER
+	@echo "Docker is not available. Please install docker"
+	@exit 1
+endif
+ifndef DOCKER_COMPOSE
+	@echo "docker-compose is not available. Please install docker-compose"
+	@exit 1
+endif
+
+# TODO: for future use, will tag docker build
+COMMIT := $(shell git rev-list -1 HEAD)
+BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+VERSION := $(shell git describe --tags --always)
+
+LEMUR_GIT_DIR = lemur-build-docker/.lemur
+
+clean: deps
+	rm -rf $(LEMUR_GIT_DIR)
+	docker-compose down -v || sudo docker-compose down -v
+
+all: deps
+	$(MAKE) copy_env_files
+	$(MAKE) lemur_checkout
+	$(MAKE) build_containers
+	$(MAKE) restart_containers
+
+copy_env_files:
+	[ -d .lemur.env ] || cp .lemur.env.dist .lemur.env && echo ".lemur.env created from scratch"
+	[ -d .pgsql.env.dist ] || cp .pgsql.env.dist .pgsql.env && echo ".pgsql.env created from scratch"
```

Comment:
I think this should be `-d .pgsql.env` right? Also, I haven't tested yet, but I think these should both be `-f` rather than `-d`?

Reply:
I make stupid bugs, fixed.  Thank you.